invenio_rdm_records/resources/config.py

```diff
@@ -48,6 +48,7 @@
 from .serializers import (
```

Comment:
minor: Add a test for at least minimal record like https://github.com/inveniosoftware/flask-resources/pull/124/files#diff-40fa549b2e04aff4adbd50658b9bfff04fb9750a75eda1cb415ef005818e258e

Reply:
See https://github.com/inveniosoftware/invenio-rdm-records/issues/1653